highfidelity/hifi
# Fix entity list search case-sensitivity and allow id search

Fixes: https://highfidelity.manuscript.com/f/cases/18386 (new since RC72)

Test Plan:
- Enter Interface in desktop mode in a domain with as many different entity types as possible
- Open Create and search with the different entity type names using both uppercase and lowercase for the first letter as well as other letters
- Verify search terms are not case-sensitive and expected matching results are shown
- Search with other queries of names, file paths, and entity IDs with varying uppercase vs lowercase
- Verify search terms are not case-sensitive, that entity IDs are valid search terms, and expected matching results are shown
